### ISSUE TITLE
Tracking domain: statistics.csob.cz

### ISSUE THREAD

**Issue by vlastikcz** [uBlockOrigin/uAssets#13540]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

https://online.csob.cz/
https://identita.csob.cz/
https://*.csob.cz/

### Describe the issue

Tracking sciripts served from the domain `statistics.csob.cz`. It might be nice to have those blocked by the privacy filter list.

### uBlock Origin version

uBlock Origin 1.42.4

### Browser name and version

Firefox 101

### Settings

 - All uBlock filters enabled
